lib/ansible/module_utils/tacp_ansible/tacp_utils.py

```diff
@@ -118,15 +118,16 @@ def get_filters_query_string(self, **kws):
 
         for k, v in kws.items():
             if isinstance(v, (list, tuple)):
-                op, *value = v
+                op, value = v[0], v[1:]
                 if op not in allowed:
                     raise Exception('Invalid operator "{}". '
                                     'Allowed: {}'.format(op, allowed))
+                if len(value) == 0:
+                    raise Exception('Must provide at least one value')
 
                 if op in ('=in=', '=out='):
-                    if len(value) > 0:
-                        value = ','.join(map(str, value))
-                        value = '({})'.format(value)
+                    value = ','.join(map(str, value))
+                    value = '({})'.format(value)
```

Comment:
I ran into an issue with filtering templates if there was a space in the template's name - the value should have "" around it in the filter to include the whole text and not cut off at a space.

Reply:
Yea I remember something similar, that's probably the reason why I initially had the third placeholder wrapped in double quotes on line 136 below.

Do you still have an example to reproduce this? If yes I suggest you create a bug and take it separately.

Let's merge this if you guys dont see any issues with OL-10845 itself